client/src/templates/Challenges/components/help-modal.tsx

```diff
@@ -45,6 +45,40 @@ const generateSearchLink = (title: string, block: string) => {
   return search;
 };
 
+interface CheckbockProps {
+  name: string;
+  i18nkey: string;
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  value: boolean;
+  href: string;
+  t: (text: string) => string;
+}
+
+function CheckboxHelpModal({
+  name,
+  i18nkey,
+  onChange,
+  value,
+  href,
+  t
+}: CheckbockProps) {
+  return (
+    <div style={{ display: 'flex', flexDirection: 'row' }}>
+      <input name={name} type='checkbox' onChange={onChange} checked={value} />
```

Comment:
```suggestion
      <input name={name} type='checkbox' onChange={onChange} checked={value} aria-label={t(i18nkey)} />
```

An input always needs a programmatic name.